### DISCUSSION TITLE
Is there any way to use `sql_query` with this?

### DISCUSSION THREAD

**Discussion by Zizico2** [weiznich/diesel_async#56] [Q&A]
I need raw sql access to get `tableoid`s in `postgresql`. Is this possible with `diesel_async`? This could even be supported in `diesel`? It's already possible to get `oid`s for values, so why not tables?

**Comment by weiznich** [comment]
Yes you can use `diesel::sql_query` with diesel async. 
For a more detailed answer please provide a minimal reproducible example of what you are trying to achieve and what does not work for you. Otherwise it's just not possible to provide more details.
